backend/docker-compose.yml

```diff
@@ -0,0 +1,137 @@
+version: "3.7"
+services:
+  rabbitmq:
+    image: rabbitmq:3.7.20-management-alpine
+    container_name: rabbitmq
+    env_file:
+      - config/config.env
+      - config/secret.env
+    expose:
+      - 5672 # internal AMQP port
+      - 15672 # internal web interface
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "1m"
+        max-file: "5"
```

Comment:
Is this for rotation? We could probably go more

Reply:
Yes this is for rotation. We can adjust this and the logging verbosity as required.